ui/app/src/nh-config/nh-global-config.ts

```diff
@@ -0,0 +1,141 @@
+import { html, css } from 'lit';
+import { contextProvided } from '@lit-labs/context';
+import { StoreSubscriber } from 'lit-svelte-stores';
+
+import { MatrixStore } from '../matrix-store';
+import { matrixContext, weGroupContext } from '../context';
+import { DnaHash, EntryHash, encodeHashToBase64 } from '@holochain/client';
+
+import { NHButton, NHComponent, NHPageHeaderCard } from '@neighbourhoods/design-system-components';
+import CreateMethod from './create-method-form';
+import CreateDimension from './create-dimension-form';
+import DimensionList from './dimension-list';
+import { query, state } from 'lit/decorators.js';
+import { b64images } from '@neighbourhoods/design-system-styles';
+
+export default class NHGlobalConfig extends NHComponent {
+  @contextProvided({ context: matrixContext, subscribe: true })
+  _matrixStore!: MatrixStore;
+
+  @contextProvided({ context: weGroupContext, subscribe: true })
+  weGroupId!: DnaHash;
+
+  @query('dimension-list')
+  _list;
+  @query('create-dimension')
+  _dimensionForm;
+
+  @state()
+  private _selectedInputDimensionRange!: Range;
+
+  @state()
+  private _inputDimensionEhs: EntryHash[] = [];
+
+  @state()
+  private _formType: "input-dimension" | "method" = "input-dimension";
+
+  _sensemakerStore = new StoreSubscriber(this, () =>
+    this._matrixStore?.sensemakerStore(this.weGroupId),
+  );
+
+  resetConfig() {
+    this._formType = "input-dimension"
+    this._list.dimensionSelected = false;
+  }
+
+  render() {
+    return html`
+      <main
+        @dimension-created=${async (e: CustomEvent) => {
```

Comment:
Please move this into a named arrow function that is part of the class. Then it can just be called here. Same thing goes for all of the inline anonymous closures.